components/tooltip/script.js

```diff
@@ -46,6 +46,14 @@ class Tooltip extends HTMLElement {
         return this.hasAttribute('async');
     }
 
+    debounce(callback, timeout = 10) {
+        this.debounceTimer;

```

Comment:
What is the purpose of this? It does nothing :D

Reply:
Removed.